### ISSUE TITLE
Missing migration in 1.2.2

### ISSUE THREAD

**Issue by Chronial** [divio/django-filer#877]
3130f010f8b25f20143 broke migrations on django 1.8 :(

```
> ./manage.py makemigrations --dry-run
Migrations for 'filer':
  0007_auto_20160629_1926.py:
    - Alter field file_ptr on image
```

```
> cat ~/venv/lib/python3.4/site-packages/filer/migrations/0007_auto_20160629_1926.py
# -*- coding: utf-8 -*-
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('filer', '0006_auto_20160623_1627'),
    ]

    operations = [
        migrations.AlterField(
            model_name='image',
            name='file_ptr',
            field=models.OneToOneField(parent_link=True, to='filer.File', auto_created=True, serialize=False, primary_key=True),
        ),
    ]
```

**Comment by bartTC** [comment]
I experience this as well, its a bit annoying as we can't do a simple `manage.py makemigrations` as it would create that migration, which then other migrations depend on, which then on a fresh deployment is missing. 🐸

**Comment by bartTC** [comment]
Actually it appears to be fixed with b21661ccd2565fe447b448f8b6e02674e6d1ff1b, it's just missing a release.

**Comment by yakky** [comment]
@bartTC filer 1.2.4 release is in process. expect it later today

**Comment by yakky** [comment]
Fixed by #880
